src/server/stream_family.cc

```diff
@@ -862,6 +975,207 @@ OpResult<vector<GroupInfo>> OpListGroups(const DbContext& db_cntx, string_view k
   return result;
 }
 
+vector<Record> streamWithRange(stream* s, streamID start, streamID end, int reverse, size_t count) {
```

Comment:
please call it `ReadRange` or some other name that starts with a verb. Also please note  our naming convention for functions

Reply:
Renamed it to `GetStreamRecords`